dev/Repeater/ItemsRepeater.idl

```diff
@@ -429,8 +429,11 @@ unsealed runtimeclass StackLayout : VirtualizingLayout
     [MUX_DEFAULT_VALUE("0.0")]
     Double Spacing { get; set; };
 
+    Boolean DisableVirtualization { get; set; };
```

Comment:
Cann you please put this API in preview ?

Reply:
I don't think he can do it because it will block NavigationView to be merged to framework package

---
In reply to: [352729386](https://github.com/microsoft/microsoft-ui-xaml/pull/1683#discussion_r352729386) [](ancestors = 352729386)

Reply:
VirtualMode seems to be something other than what we are trying to do here - so it will be confusing to name it that. This is not a listview or repeater thing. It is specific to the layout.

Reply:
you do not need the api to be public to consume it within mux no ?

Reply:
You do not. Please use the PREVIEW attribute until it's been API reviewed. Until then it should work fine in tests.

---
In reply to: [352744316](https://github.com/microsoft/microsoft-ui-xaml/pull/1683#discussion_r352744316) [](ancestors = 352744316)